### DISCUSSION TITLE
有關print和str指令的問題

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#56] [Q&A]
我的想法是,把abc快速對調後,轉換成字串再列印出來,但是失敗了請問這樣寫是因為print還是str指令的限制,還是有別的解釋方法嗎?
我還是個萌新,麻煩大家了!!
![python 問題](https://user-images.githubusercontent.com/91265127/136798468-6cfb61a9-369a-48bc-bffc-7b980c4683ef.PNG)

**Comment by jacky0115** [comment]
是因為str()的原因喔，你不行在str()裡面對調資料
可以試試把 **對調資料** 和 **印出字串** 這兩部分分兩行寫
如果你想要轉成字串再印出來可以用 `print(str(a),str(b),str(c))`

**Reply by kaoyihua27** [reply]
原來如此!謝謝你囉!
